ipaserver/plugins/groupfas.py

```diff
@@ -17,30 +17,40 @@
 from ipaserver.plugins.group import group_show
 from ipaserver.plugins.internal import i18n_messages
 
-group.possible_objectclasses.append("fasgroup")
-if "objectclass" not in group.default_attributes:
-    group.default_attributes.append("objectclass")
+from .fasutils import URL
 
-default_attributes = [
+if "fasgroup" not in group.possible_objectclasses:
+    group.possible_objectclasses.append("fasgroup")
+
+group_fas_attributes = [
     "fasurl",
     "fasmailinglist",
     "fasircchannel",
 ]
-group.default_attributes.extend(default_attributes)
+group.default_attributes.extend(group_fas_attributes)
+# always fetch objectclass so group_show can show fasgroup property
+group.default_attributes.append("objectclass")
```

Comment:
This used to be present above but protected with a presence test, is it not needed?